pkg/controller.v1/tensorflow/job.go

```diff
@@ -26,16 +24,6 @@ const (
 	SuccessfulDeleteJobReason = "SuccessfulDeleteJob"
 )
 
-var (
```

Comment:
If we don't remove it, the metrics will have conflict names, right?

Reply:
correct and hence had to remove it. are we still planning to use this file in future?

Reply:
No. They are not being used. It's fine to remove definition from legacy files

Reply:
https://github.com/kubeflow/tf-operator/issues/1367